setup.py

```diff
@@ -14,6 +14,7 @@
         "numpy",
         'importlib_resources; python_version < "3.7"',
         "isodate",
+        "pytest",
```

Comment:
pytest should not be a library dependency.

Reply:
This helped us integrate well with pycharm IDE. Will be happy to remove if you want.